Update the full example config

sosthene-nitrokey opened the pull request:
Take the latest modifications from docs.nitrokey.com Add information on how to login through `C_Login`

sosthene-nitrokey committed:
Update the full example config

Take the latest modifications from docs.nitrokey.com
Add information on how to login through `C_Login`

jans23 commented on p11nethsm.example.conf:
```diff
@@ -38,9 +38,18 @@ slots:
+      #
+      # If the field is not provided, it is expected to be given through `C_Login` function with the
+      # `CKU_USER` userType
       password: "localpass"
```
Maybe by default username and password should be commented (inactive) because the default values will be wrong for most users.

jans23 replied:
Is this really changed as intended? To me it looks mixed up: First password, then username than passwords are explained again.

![grafik](https://github.com/user-attachments/assets/33f4877b-2a9a-4cfa-b97c-eb9488800c59)

sosthene-nitrokey committed:
Make the password be absent by default

Since in all cases the default password is not valid, remove it from the documentation
See https://github.com/Nitrokey/nethsm-pkcs11/pull/245#discussion_r2143017494
